app/index.ts

```diff
@@ -1,5 +1,7 @@
-import { Tracker, UserFactory } from "@relewise/client";
+// import { Tracker, UserFactory } from "@relewise/client";
 
-const tracker = new Tracker("...", "...");
+// const tracker = new Tracker("...", "...");
 
-tracker.tractProductView({ productId: "p-1", user: UserFactory.anonymous() });
\ No newline at end of file
+// tracker.tractProductView({ productId: "p-1", user: UserFactory.anonymous() });
```

Comment:
er det meningen at der skal være udkommenteret kode her?

Reply:
Ja det er lidt forvirrende men `./app` er en mere en playground, hvor man kan teste op i mod SDK'et og få highligting på.
Jeg tænker om vi skal rename den `./app` til `./playground`.

Hvad tænker du @RAORelewise ?

Reply:
Har ikke nogen holdning til navngivningen, men at hvis man læser vores readme så er det åbenlyst hvad /app projektet er til for. Så synes ikke det gør så meget der er udkommenteret kode i, det er jo et "test" projekt

Reply:
👍